src/styles/main.scss

```diff
@@ -2,3 +2,213 @@
 @import "partials/partials";
 @import "elements/elements";
 
+$primary-color: rgba(0, 51, 103, 1);
+
+/* Space out content a bit */
+
+body {
+  padding-top: 49px;
+}
+
+/* Everything but the jumbotron gets side spacing for mobile first views */
+
+.header, .footer {
+  padding-right: 15px;
+  padding-left: 15px;
+}
+
+/* Custom page header */
+
+.header {
+  padding-bottom: 20px;
+  border-bottom: 1px solid #e5e5e5;
+}
+
+/* Make the masthead heading the same height as the navigation */
+
+.header h3 {
+  margin-top: 0;
+  margin-bottom: 0;
+  line-height: 40px;
+}
+
+/* Custom page footer */
+
+.footer {
+  padding: 20px 30px;
+  font-size: 18px;
+  color: #777;
+  border-top: 3px solid #ddd;
+  background: #e1e1e1;
+}
+.navbar {
+  background: $primary-color;
+}
+.navbar img {
+  margin-top: -7px;
+}
+.index-infobar {
+  background: #e1e1e1;
```

Comment:
Can we move this into a variable? I would suggest having a scss file just for variables that we are using across the site?